app/views/enquiries/index.html.erb

```diff
@@ -1,18 +1,46 @@
-<h1>Enquiries</h1>
-<% if @enquiries.present? %>
-  <% @enquiries.each do |e| %>
-    <div class="row">
-      <div class="col-sm-4">
-        <%= e.created_at %>
-      </div>
-      <div class="col-sm-8">
-        <%= e.first_name %> <%= e.last_name %>
-      </div>
-      <div class="col-sm-2">
-        <%= link_to "Edit", edit_enquiry_path(e) %>
-      </div>
-    </div>
-  <% end %>
-<% else %>
+<div class="row">
+  <div class="col-sm-12 text-xs-center bootstrap-welcome">
+    <h1 class="jumbotron">Referrals</h1>
+    <p class="lead">Referral Reference System</p>
+  </div>
+</div>
+
+<div>
+  <div class>
+    <%= link_to "New Enquiry", new_enquiry_path, class: 'button' %>
+  </div>
+</div>
+
+<div>
+  <% if @enquiries.present? %>
+  <h2>Your Referrals</h2>
+
+  <table style="width: 100%">
```

Comment:
We want to avoid using tables since they are rubbish on mobile. I've started to use bootstrap rows / cols instead

Reply:
ok fair point I'll keep that in mind with future styling.